### ISSUE TITLE
0.4.5 Error - Cannot read properties of undefined (reading 'subtle')

### ISSUE THREAD

**Issue by v-moravec** [neondatabase/serverless#19]
## Steps to reproduce
npx nuxi init \<project-name\>
yarn add drizzle-orm @neondatabase/serverless
yarn add -D drizzle-kit

db.ts

```
import {neonConfig, Pool} from '@neondatabase/serverless';
import {drizzle, NeonDatabase} from 'drizzle-orm/neon-serverless';
import * as schema from '~/server/drizzle/schema';

// if we're running locally
if (!process.env.VERCEL_ENV) {
  // Set the WebSocket proxy to work with the local instance
  neonConfig.wsProxy = (host) => `${host}:5433/v1`
  // Disable all authentication and encryption
  neonConfig.useSecureWebSocket = false
  neonConfig.pipelineTLS = false
  neonConfig.pipelineConnect = false
}

const config = useRuntimeConfig()

if (!config.dbUrl) {
  throw new Error('POSTGRES_URL is not set')
}


let db: NeonDatabase<typeof schema> | undefined = undefined

export const useDB = () => {
  if (db) {
    return db
  }
  const pool = new Pool({ connectionString: config.dbUrl })
  db = drizzle(pool, { schema })
  return db
}

db = useDB()

```
## Expected result
Should work

## Actual result
Instead throws error

## Environment
Nuxt 3.5
@neondatabase/serverless 0.4.5
Node v18.12.0

## Logs, links
ERROR  [worker reload] [worker init] Cannot read properties of undefined (reading 'subtle')                                                                             3:07:35 PM

  at node_modules/@neondatabase/serverless/index.js:1:59242
  at node_modules/@neondatabase/serverless/index.js:1:347
  at node_modules/@neondatabase/serverless/index.js:1:60107
  at node_modules/@neondatabase/serverless/index.js:1:347
  at node_modules/@neondatabase/serverless/index.js:46:2299
  at node_modules/@neondatabase/serverless/index.js:1:347
  at node_modules/@neondatabase/serverless/index.js:46:25042
  at node_modules/@neondatabase/serverless/index.js:1:347
  at Object.<anonymous> (node_modules/@neondatabase/serverless/index.js:46:25975)
  at Module._compile (node:internal/modules/cjs/loader:1159:14)

**Comment by jawj** [comment]
Sounds like the driver is trying to access SubtleCrypto in a context where it's not available. 

* If you have `?sslmode=require` at the end of your connection string, are you able to take that off (e.g. if connecting to a local DB in development)?
* Can you use Node 20 instead of Node 18 (where `crypto.subtle` is defined without needing to import anything)?
* Can you run Node 18 with the `--experimental-global-webcrypto` flag?
* Can you import `crypto` as a global before you dynamically import the driver? That would look something like:

```
import crypto from "crypto";
globalThis.crypto = crypto;
const { Pool } = await import('@neondatabase/serverless');
```

**Comment by v-moravec** [comment]
Well, it seems to be fixed with 0.4.9 (also works in 0.4.4 and lower). However with 0.4.9 I'm getting this now on Cloudflare Pages (using Workers under the hood) -  `Uncaught TypeError: Pool is not a constructor`, locally it works ok with `yarn dev`, but not with building it with `wrangler and wrangler pages dev dist/`.

**Comment by jawj** [comment]
OK, good to hear. I think changes in the underlying pg package, which I'd previously omitted to pin to a specific version, must have been responsible for the earlier issues, then.

I've not encountered this new error before. Are you able to provide a simple repro?

**Comment by jawj** [comment]
I've just released 0.4.10, which I believe will fix your import problem. Please do let me know either way.

**Comment by v-moravec** [comment]
I can confirm everything works as expected now.
